Libraries/LibWeb/TrustedTypes/TrustedTypePolicyFactory.cpp

```diff
@@ -154,6 +155,12 @@ bool TrustedTypePolicyFactory::is_html(const JS::Value& value)
     return value.is_object() && is<TrustedHTML>(value.as_object()) && as<TrustedHTML>(value.as_object()).data_is_set();
 }
 
+// https://w3c.github.io/trusted-types/dist/spec/#dom-trustedtypepolicyfactory-isscript
+bool TrustedTypePolicyFactory::is_script(const JS::Value& value)
```

Comment:
wrong direction for const (east vs west). but JS::Value should be passed by value anyway. it's pointer-sized.